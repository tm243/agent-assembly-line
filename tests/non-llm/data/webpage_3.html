<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <title>TOTALLY LEGIT BANK™ - Your Money Is Definitely Safe With Us</title>
  <style>
    body {
      font-family: 'Comic Sans MS', cursive;
      margin: 0;
      padding: 0;
      background-color: #ffd700;
      color: #8b008b;
    }

    header {
      background-color: #ff69b4;
      padding: 20px;
      text-align: center;
    }

    .container {
      width: 80%;
      margin: 0 auto;
      padding: 20px;
      background-color: rgba(255, 255, 255, 0.7);
      border-radius: 15px;
      box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
    }
  </style>
</head>

<body>

  <div class="moduleContainer">

    <div data-cont-id="123456" data-module-name="FoosModule">
      <div class="relative shadow-modules foo:shd-n container">
        <div data-comp="FoosModule" class="block jd-module px-4 foo:px-6 py-6">
          <section data-layout-name="FreeHorizontalLayout" data-comp="FooList"
            class="grid gap-6 foo:gap-y-12 foo:auto-cols-fr foo:grid-flow-col">
            <article data-comp="FooCont" data-cid="12345678" data-cont-id="123456"
              class="group" data-foo-name="VerticalOverlapFoo" data-what="12345678_o"
              data-so="0-6" data-so-idx="6" data-ever="data-ever-value"><a
                index="0" data-comp="FooLinkCont"
                href="/bar/some-article.html"
                class="group foo:realtive relative">
                <figure data-comp="Image" class="mb-4 foo:w-full">
                    <a-img width="1920" height="1079"
                      src="https://placeholder.example/imgs/placeholder-image.jpeg"
                      style="aspect-ratio: 1920 / 1079;">
                      <img
                        src="https://placeholder.example/imgs/placeholder-image.jpeg" width="1920" height="1079"
                        style="aspect-ratio: 1920 / 1079;" />
                    </a-img>
                  </figure>
              </a>
              <div
                class="foo:relative foo:mx-4 foo:-mt-12 foo:px-4 foo:pt-4  relative mx-2 -mt-12 px-2 pt-4 foo:px-4">
                <a index="0" data-comp="FooLinkCont"
                  href="/bar/some-article.html" class="group"
                   >
                  <header data-comp="FooHeader">
                    <h3 class="flex flex-col">
                      <span data-comp="FooKicker"
                        class="flex flex-wrap items-center gap-2 text-base leading-none dark:text-gray-100 mb-2">Digitizing
                        Memories</span>
                        <span data-comp="FooHeadline"
                          class="text-xl foo:text-2xl max-w-prose font-bold group-hover:text-brand-branding dark:group-hover:text-white">
                          <span data-upscore-title="true">Restoring old VHS tapes with AI technology</span>
                      </span>
                    </h3>
                  </header>
                  <p data-comp="FooSynopsis" class="mt-3 text-base">VHS tapes and players gradually deteriorate in
                    attics – and memories fade with them. Here's how to save your recordings for the digital age.</p>
                  <footer data-comp="FooMeta"
                    class="flex flex-wrap items-center gap-3 leading-none text-sm mt-4">
                    <svg class="placeholder-svg" xmlns="http://www.w3.org/2000/svg"></svg>
                    <span class="flex items-center">
                      <svg class="placeholder-svg" xmlns="http://www.w3.org/2000/svg"></svg>
                      123
                    </span>
                    <div class="flex gap-1">
                      <span data-comp="Branding" class="flex whitespace-nowrap leading-none">
                        <span class="mr-2 inline-block" style="width:0;"></span>
                        example.com
                      </span>
                    </div>
                  </footer>
                </a>
              </div>
              <div data-what="12345678" data-so="456" data-ever="123"></div>
            </article>
            <hr class="border-gray-100 dark:border-gray-600 foo:hidden">
            <article data-comp="FooCont" data-cid="123456" data-cont-id="123456"
              class="group mx-2 px-2 foo:m-0 foo:p-0" data-foo-name="VerticalOverlapFoo"
              data-what="123456_o" data-so="0-8" data-so-idx="8"
              data-ever="data-ever-value"><a index="1" data-comp="FooLinkCont"
                href="/bar/other-article.html" class="group foo:realtive">
                <figure data-comp="Image"
                  class="mb-4 relative z-10 mb-4 foo:z-auto float-right w-[28%] ml-4 foo:float-none foo:ml-0 foo:w-full">
                  <a-img width="2001" height="1124" src="https://placeholder.example/imgs/placeholder-image.jpeg"
                    style="aspect-ratio: 2001 / 1124; --aspect-ratio-width: 2001; --aspect-ratio-height: 1124;"><img
                      src="https://placeholder.example/imgs/placeholder-image.jpeg" width="2001" height="1124"
                      style="aspect-ratio: 2001 / 1124;"></a-img>
                </figure>
              </a>
              <div class="foo:relative foo:mx-4 foo:-mt-12 foo:px-4 foo:pt-4"><a index="1"
                  data-comp="FooLinkCont"
                  href="/bar/other-article.html" class="group">
                  <header data-comp="FooHeader">
                    <h3 class="flex flex-col"><span data-comp="FooKicker"
                        class="flex flex-wrap items-center gap-2 text-base leading-none dark:text-gray-100 mb-2">Study
                        with Gaps</span><span data-comp="FooHeadline"
                        class="text-xl foo:text-2xl max-w-prose font-bold group-hover:text-brand-branding dark:group-hover:text-white"><span
                          data-upscore-title="true">The surveillance assessment operates with numerous
                          unknowns</span></span></h3>
                  </header>
                  <p data-comp="FooSynopsis" class="mt-3 text-base hidden foo:block">A study for the
                    government was intended to calculate the extent of state surveillance. Not all information was
                    made available to researchers.</p>
                  <footer data-comp="FooMeta"
                    class="flex flex-wrap items-center gap-3 leading-none text-sm mt-4"><span
                      class="flex items-center ">
                      <svg xmlns="http://www.w3.org/2000/svg" viewBox=""></svg>
                      A</span>
                  </footer>
                </a></div>
              <div data-what="123456" data-so="0-9" data-so-idx="9"
                data-ever="data-ever-value"></div>
            </article>
          </section>
        </div>
      </div>
    </div>

  </div>

</body>

</html>