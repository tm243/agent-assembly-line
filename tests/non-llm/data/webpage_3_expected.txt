# TOTALLY LEGIT BANK™ - Your Money Is Definitely Safe With Us

## Digitizing
                        Memories.  
VHS tapes and players gradually deteriorate in
                    attics – and memories fade with them. Here's how to save your recordings for the digital age.
[Read more](file://tests/bar/some-article.html)

## Study
                        with Gaps.  
A study for the
                    government was intended to calculate the extent of state surveillance. Not all information was
                    made available to researchers.
[Read more](file://tests/bar/other-article.html)

